packit_service/worker/helpers/build/copr_build.py

```diff
@@ -569,6 +574,26 @@ def handle_build_submit_error(self, ex):
         """
         Handle errors when submitting Copr build.
         """
+        if (
+            # don't retry the task if user action needed
+            not (
+                MISSING_PERMISSIONS_TO_BUILD_IN_COPR in str(ex)
+                or NOT_ALLOWED_TO_BUILD_IN_COPR in str(ex)
+            )
+            and not self.celery_task.is_last_try()
```

Comment:
Shouldn't we on the last retry notify the user through an issue if an issue repository is configured? Similarly to propose-downstream/bodhi.

In my implementation, I refactored `notify_user_about_failure` into `JobHandler` and made it a bit more general like so:

```python
    def notify_user_about_failure(self, title: str, body: str) -> None:
        """
        If user configures `issue_repository`,
        Packit will create there an issue with the details.
        If the issue already exists and is opened, comment will be added
        instead of creating a new issue.

        The issue will be a place where to re-trigger the job.

        Args:
            title: Title of the issue to create.
            body: Content to be sent to the user (comment or issue description,
                for issue description a contact footer is added).
        """
        if not self.job_config.issue_repository:
            logger.debug(
                "No issue repository configured. User will not be notified about the failure."
            )
            return

        logger.debug(
            f"Issue repository configured. We will create "
            f"a new issue in {self.job_config.issue_repository}"
            "or update the existing one."
        )
        issue_repo = self.service_config.get_project(
            url=self.job_config.issue_repository
        )
        PackageConfigGetter.create_issue_if_needed(
            project=issue_repo,
            title=title,
            message=body
                    + f"\n\n---\n\n*Get in [touch with us]({CONTACTS_URL}) if you need some help.*",
            comment_to_existing=body,
        )
```